pkg/sql/schemachanger/scbuild/dependencies.go

```diff
@@ -185,19 +185,12 @@ type AstFormatter interface {
 	FormatAstAsRedactableString(statement tree.Statement, annotations *tree.Annotations) redact.RedactableString
 }
 
-// CommentCache represent an interface to fetch and cache comments for
-// descriptors.
-type CommentCache interface {
-	scdecomp.CommentGetter
-
-	// LoadCommentsForObjects explicitly loads commentCache into the cache give a list
-	// of object id of a descriptor type.
-	LoadCommentsForObjects(ctx context.Context, objIDs []descpb.ID) error
-}
-
-// ZoneConfigGetter see scdecomp.ZoneConfigGetter
+// ZoneConfigGetter see scdecomp.ZoneConfigGetter.
 type ZoneConfigGetter scdecomp.ZoneConfigGetter
 
+// CommentGetter see scdecomp.CommentGetter.
+type CommentGetter scdecomp.CommentGetter
+
```

Comment:
We probably need an interface here which embeds `ZoneConfigGetter`, `CommentGetter` and has a method to do the equivalent of the `lookupFn` passed to `WalkDescriptor`. This might be out of scope of this PR, you be the judge.